logger/src/commonMain/kotlin/com/wire/kalium/logger/ObfuscateUtil.kt

```diff
@@ -0,0 +1,82 @@
+@file:Suppress("MaximumLineLength", "MagicNumber", "ReturnCount", "MaxLineLength", "LongMethod")
+
+package com.wire.kalium.logger
+
+fun obfuscateLogMessage(message: String): String {
+    val ipV4 = ".*\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}.*".toRegex()
+    val ipV4Exact = "\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}".toRegex()
+
+    val password = ".*\"password\":\\s*\"([^\"]*)\".*".toRegex()
+    val passwordExact = "\"password\":\\s*\"([^\"]*)\"".toRegex()
+
+    val ipV6 =
+        ".*(?:^|(?<=\\s))(([0-9a-fA-F]{1,4}:){7,7}[0-9a-fA-F]{1,4}|([0-9a-fA-F]{1,4}:){1,7}:|([0-9a-fA-F]{1,4}:){1,6}:[0-9a-fA-F]{1,4}|([0-9a-fA-F]{1,4}:){1,5}(:[0-9a-fA-F]{1,4}){1,2}|([0-9a-fA-F]{1,4}:){1,4}(:[0-9a-fA-F]{1,4}){1,3}|([0-9a-fA-F]{1,4}:){1,3}(:[0-9a-fA-F]{1,4}){1,4}|([0-9a-fA-F]{1,4}:){1,2}(:[0-9a-fA-F]{1,4}){1,5}|[0-9a-fA-F]{1,4}:((:[0-9a-fA-F]{1,4}){1,6})|:((:[0-9a-fA-F]{1,4}){1,7}|:)|fe80:(:[0-9a-fA-F]{0,4}){0,4}%[0-9a-zA-Z]{1,}|::(ffff(:0{1,4}){0,1}:){0,1}((25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])\\.){3,3}(25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])|([0-9a-fA-F]{1,4}:){1,4}:((25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])\\.){3,3}(25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9]))(?=\\s|\$).*".toRegex()
```

Comment:
Can use string literals to avoid the annoying escaping of strings?

For example:
```kt
val password = """.*"password":\s*"([^\"]*)".*""".toRegex()
```

instead of:
```kotlin
val password = ".*\"password\":\\s*\"([^\"]*)\".*".toRegex()
```

This makes it easier also to copy/paste from/to Regex analysers, without the added back slashes `\`.